session.go

```diff
@@ -122,30 +123,16 @@ func (s *Session) Passive(nw string) error {
 		s.Data.Close()
 		s.Data = nil
 	}
-	li, err := s.Server.listen(nw, s.passiveHost()+":0")
+	addr := net.JoinHostPort(s.Server.Host, "0")
```

Comment:
That seems reasonable. I might do away with `Server.Host` altogether, since a lot of clients ignore the IP in a PASV reply, and the client is guaranteed to be able to the server at the control channel IP.